scene/3d/skeleton_3d.cpp

```diff
@@ -953,6 +953,63 @@ void Skeleton3D::_skin_changed() {
 	_make_dirty();
 }
 
+Vector<int> Skeleton3D::get_process_order() {
```

Comment:
I would prefer if this was written using get_parentless_bones and get_children (though it would be recursive, I think that's ok)